testdata/project-v4-multigroup/config/crd/kustomization.yaml

```diff
@@ -19,12 +19,12 @@ resources:
 patches:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
 # patches here are for enabling the conversion webhook for each CRD
-#- path: patches/webhook_in_crew_captains.yaml
-#- path: patches/webhook_in_ship_frigates.yaml
-#- path: patches/webhook_in_ship_destroyers.yaml
-#- path: patches/webhook_in_ship_cruisers.yaml
-#- path: patches/webhook_in_sea-creatures_krakens.yaml
-#- path: patches/webhook_in_sea-creatures_leviathans.yaml
+- path: patches/webhook_in_crew_captains.yaml
+- path: patches/webhook_in_ship_frigates.yaml
+- path: patches/webhook_in_ship_destroyers.yaml
+- path: patches/webhook_in_ship_cruisers.yaml
+- path: patches/webhook_in_sea-creatures_krakens.yaml
+- path: patches/webhook_in_sea-creatures_leviathans.yaml
 #- path: patches/webhook_in_foo.policy_healthcheckpolicies.yaml
```

Comment:
TODO:
we need to check why when is multigroup those are not uncommenting as well

Reply:
Hi @Sajiyah-Salat,

> Also, as you have said, the issue is out of the scope of this PR. Could you let me know what we can address in a follow-up PR?
> Regards: [PR Discussion](https://github.com/kubernetes-sigs/kubebuilder/pull/3629#discussion_r1328065993)

In the sample project which is using a multi-group layout, we can see that some patches were not uncommented.
Why does this happen? 
Because those patches should not exist in the first place.
They have been created when we run `create api`, but should only be created in specific scenarios, those when we run `webhook create`. The change of this PR is in the webhook subcommands, therefore, it is not reflected in this case scenario.

However, since the bug found is not part of the context of this PR, I would like to refer to the issue created. I think there is a better place to discuss this. https://github.com/kubernetes-sigs/kubebuilder/issues/3630